src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -210,13 +230,51 @@ public static ColumnCondition condition(ColumnMetadata column, FieldIdentifier u
         return new UDTFieldCondition(column, udtField, op, terms);
     }
 
+    /*
+     * Don't change or remove entries (and ordinals) or it will break cross version compatibility
+     * as the ordinals are sent in messages and stored by Accord for in flight transactions
+     */
```

Comment:
If it's really important, why not just give each kind an explicit identifier, sort of like what we have in `cql3.Operator`?

Reply:
I changed things up a bit and now it uses explicit IDs.